sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.institutions.controller.ts

```diff
@@ -139,6 +140,8 @@ export class ApplicationOfferingChangeRequestInstitutionsController extends Base
       programId: application.currentAssessment.offering.educationProgram.id,
       isProgramActive:
         application.currentAssessment.offering.educationProgram.isActive,
+      isProgramExpired:
```

Comment:
Minor, suggestion, creating a const to `application.currentAssessment.offering` can make this block easier to read.